onyo/lib/tests/test_commands_mv.py

```diff
@@ -69,12 +70,79 @@ def test_onyo_mv_into_self(inventory: Inventory) -> None:
                   destination=asset_path.parent,
                   message="some subject\n\nAnd a body")
 
+    # source does not exist
+    pytest.raises(ValueError,
+                  onyo_mv,
+                  inventory,
+                  source=inventory.root / "not-existent",
+                  destination=dir_path,
+                  message="some subject\n\nAnd a body")
+
+
+@pytest.mark.ui({'yes': True})
+def test_onyo_mv_errors_before_mv(inventory: Inventory) -> None:
+    """`onyo_mv` must raise the correct error and is not allowed to move/commit anything, if one of
+    the sources does not exist.
+    """
+    asset_path = inventory.root / "somewhere" / "nested" / "TYPE_MAKER_MODEL.SERIAL"
+    destination_path = inventory.root / 'empty'
+    old_hexsha = inventory.repo.git.get_hexsha()
+
+    # one of multiple sources does not exist
+    pytest.raises(ValueError,
+                  onyo_mv,
+                  inventory,
+                  source=[asset_path, inventory.root / "not-existent"],
+                  destination=destination_path,
+                  message="some subject\n\nAnd a body")
+
+    # nothing was moved and no new commit was created
+    assert asset_path.is_file()
+    assert not (destination_path / asset_path.name).is_file()
+    # git is correct
+    assert inventory.repo.git.get_hexsha() == old_hexsha
+    # TODO: verifying cleanness of worktree does not work,
+    #       because fixture returns inventory with untracked stuff
+    # assert inventory.repo.git.is_clean_worktree()
+
+
+@pytest.mark.skip(reason="still a BUG: #516")
+@pytest.mark.ui({'yes': True})
+@pytest.mark.repo_dirs("a/b/c", "a/d/c")
+def test_onyo_mv_possible_error(inventory: Inventory) -> None:
```

Comment:
I have trouble understanding the name of this test and how it relates to what' supposed to be tested here.

Also: An alternative to skipping would be to achieve the outcome with the current implementation. We have the issue and addressing it would simply involve a change in the test. So, if you make the target explicit (`a/d/c/c`) the test should pass and reflects the implemented logic.

Reply:
Hm, I can see that, but prefer the current way. The "explicit target" test could be added additionally, but I find it more helpful to already have a test that I can use to verify when I later try to fix the underlying bug.

That said -- the name of the test is a temporary one, and I'll adjust it. I just forgot to change it from before -- oops.